waffle-doppelganger/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@ethereum-waffle/doppelganger",
+  "description": "Mock smart contracts in a smart way.",
+  "version": "2.4.1",
```

Comment:
Are we sure this is the final api? I'd go with `0.1.0` for now, since we probably will do some breaking changes to this module